patches/chrome-android-java-res-menu-main_menu.xml.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/android/java/res/menu/main_menu.xml b/chrome/android/java/res/menu/main_menu.xml
+index 4580a99b4e573076e5668eb71745ba18332e17a8..c7e33f38aacc4853ec168a7cda509174451d4170 100644
+--- a/chrome/android/java/res/menu/main_menu.xml
++++ b/chrome/android/java/res/menu/main_menu.xml
+@@ -80,6 +80,7 @@
+             android:title="@string/menu_help" />
+         <item android:id="@+id/enter_vr_id"
+             android:title="@string/enter_vr" />
++        <item android:id="@+id/exit_id" android:title="@string/menu_exit" />
```

Comment:
can you use an `include` here to make this patch extensible?

Reply:
For the record, `include` tag is not supported by menu xml resource. (layout support it)